json_logic/__init__.py

```diff
@@ -1,203 +1,982 @@
-# This is a Python implementation of the following jsonLogic JS library:
-# https://github.com/jwadhams/json-logic-js
-from __future__ import unicode_literals
+"""
+This is a Python implementation of the JsonLogic JS library:
+https://github.com/jwadhams/json-logic-js
+"""
 
-import sys
-from six.moves import reduce
-import logging
-
-logger = logging.getLogger(__name__)
 
+# Python 2 fallbacks
+from __future__ import division, unicode_literals
 try:
-    unicode
+    str = unicode
```

Comment:
As you've already pulled in `six`, is there any objection to using `six.text_type` throughout the file in place of `str`?

Reply:
Thanks for suggestion.

Made some changes. Please verify if that's what you expected.